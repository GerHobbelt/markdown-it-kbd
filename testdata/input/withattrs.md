# Test

This combination is cool and has custom data: [[alt]]{data-custom=foo}+[[f4]]{data-custom=bar}.

This combination is very important: [[ctrl]]{.important}+[[v]]{.important}.
