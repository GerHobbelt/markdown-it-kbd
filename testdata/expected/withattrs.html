<h1>Test</h1>
<p>This combination is cool and has custom data: <kbd data-custom="foo">alt</kbd>+<kbd data-custom="bar">f4</kbd>.</p>
<p>This combination is very important: <kbd class="important">ctrl</kbd>+<kbd class="important">v</kbd>.</p>
